core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/common/Status.java

```diff
@@ -50,7 +50,11 @@
     EXTERNAL_DATABASE_START_FAILED(StatusKind.FINAL),
     EXTERNAL_DATABASE_STOP_IN_PROGRESS(StatusKind.PROGRESS),
     EXTERNAL_DATABASE_STOP_FINISHED(StatusKind.PROGRESS),
-    EXTERNAL_DATABASE_STOP_FAILED(StatusKind.FINAL);
+    EXTERNAL_DATABASE_STOP_FAILED(StatusKind.FINAL),
+    LOAD_BALANCER_UPDATE_IN_PROGRESS(StatusKind.PROGRESS),
+    LOAD_BALANCER_UPDATE_FINISHED(StatusKind.FINAL),
```

Comment:
In DetailedStackStatus.java:
LOAD_BALANCER_UPDATE_FINISHED(Status.AVAILABLE)

It is AVAILABLE not LOAD_BALANCER_UPDATE_FINISHED and it is correct. I think we should remove LOAD_BALANCER_UPDATE_FINISHED.

Reply:
Hi @sodre90. This PR was closed 2.5 months ago. I get your point, but it's a little late to make changes to it now. Can I ask what prompted this?